styles/footer.css

```diff
@@ -0,0 +1,38 @@
+.footer {
+  position: relative;
+  bottom: 201px;
+}
+
+.footer__wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.footer-socials {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.footer__copyright {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.footer__item {
+  margin-left: 30px;
+}
+.footer__item:last-child {
+  margin-right: 0;
+}
+.copyright__text {
+  margin-right: 30px;
+  font-size: 16px;
+  font-family: "Arial";
```

Comment:
Add font family at the end: `"Arial", sans-serif`

Reply:
fixed